### ISSUE TITLE
Preliminary Release

### ISSUE THREAD

**Issue by solomonhawk** [vigetlabs/microcosm-devtools#26]
I'm interested in getting to a 1.0 (or other initial release). This may be a ways off and we may need to start by discussing what possible features to include and then pare that down to the minimal useful set of features that we need before we start publicizing this project.

What feature-set is in-scope for the preliminary release?
What steps are there to get from where we are to that release?

Steps:
- [ ] Add OSS stuff to the repo (changelog, contributing, license)
- [ ] Determine release feature-set, create and assign issues
- [ ] Do the work
- [ ] Release?

For @nhunzaker What would you ideally like to see in the initial release? What would be some stretch goals?

**Comment by nhunzaker** [comment]
Pinging @efatsi and @prayasht on this too.

I think we are to a point where we could release a 0.5 version. Everything is together, the tools just don't do much.

Let's create issues for things that we want to get done. Then let's figure out what can fit into a 1.0.

I'm also happy to host a meeting if you'd like to get together as a group.

**Comment by efatsi** [comment]
👍 i'm down for a little meeting, or just to throw in issues willy nilly and then refine them together later

**Comment by prayash** [comment]
A meeting would be great!

**Comment by nhunzaker** [comment]
I'll schedule a meeting. In the meantime, please file anything you want to get in.

**Comment by nhunzaker** [comment]
Meeting scheduled.
